src/NzbDrone.Core/Indexers/TitansOfTv/TitansOfTvSettings.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using FluentValidation;
+using FluentValidation.Results;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Indexers.TitansOfTv
+{
+
+       public class TitansOfTvSettingsValidator : AbstractValidator<TitansOfTvSettings>
+    {
+        public TitansOfTvSettingsValidator()
+        {
+          //  RuleFor(c => c.ApiKey).ValidRootUrl();
+        }
+    }
+
+    public class TitansOfTvSettings : IProviderConfig
+    {
+     private static readonly TitansOfTvSettingsValidator Validator = new TitansOfTvSettingsValidator();
+
+     public TitansOfTvSettings()
```

Comment:
Ya, didn't know if they would be needed for serialization or something. Ill remove it